{"uuid": "4bdcb66e-c836-4799-ad57-448cd8e69965", "counter": 5, "stdout": "\u001b[1;31mfatal: [demo03]: UNREACHABLE! => {\"changed\": false, \"msg\": \"Failed to connect to the host via ssh: Warning: Permanently added '10.100.0.121' (ED25519) to the list of known hosts.\\r\\nLoad key \\\"/tmp/tmpp8zlk0hi\\\": error in libcrypto\\r\\nuser_diaken@10.100.0.121: Permission denied (publickey,gssapi-keyex,gssapi-with-mic,password).\", \"unreachable\": true}\u001b[0m", "start_line": 4, "end_line": 5, "runner_ident": "cda10824-2480-4562-9c43-e7e310b4118d", "event": "runner_on_unreachable", "pid": 231003, "created": "2025-05-17T23:36:49.361847+00:00", "parent_uuid": "005056b7-f67d-4a7a-7f41-000000000010", "event_data": {"playbook": "/tmp/tmpc67ubrr6.yml", "playbook_uuid": "75a05ebb-8558-459c-8451-85a5f1f6f6a9", "play": "Revisar logs del sistema y autenticaci\u00f3n", "play_uuid": "005056b7-f67d-4a7a-7f41-000000000002", "play_pattern": "target_host", "task": "Gathering Facts", "task_uuid": "005056b7-f67d-4a7a-7f41-000000000010", "task_action": "gather_facts", "resolved_action": "ansible.builtin.gather_facts", "task_args": "", "task_path": "/tmp/tmpc67ubrr6.yml:2", "host": "demo03", "remote_addr": "demo03", "start": "2025-05-17T23:36:49.227520+00:00", "end": "2025-05-17T23:36:49.361403+00:00", "duration": 0.133883, "res": {"unreachable": true, "msg": "Failed to connect to the host via ssh: Warning: Permanently added '10.100.0.121' (ED25519) to the list of known hosts.\r\nLoad key \"/tmp/tmpp8zlk0hi\": error in libcrypto\r\nuser_diaken@10.100.0.121: Permission denied (publickey,gssapi-keyex,gssapi-with-mic,password).", "changed": false}, "uuid": "4bdcb66e-c836-4799-ad57-448cd8e69965"}}